packages/plugin-http-client/src/rest/mapper/EntityListMapper.ts

```diff
@@ -0,0 +1,36 @@
+import { BaseMapper } from './BaseMapper';
+import { EntityMapper } from './EntityMapper';
+import { EntityConstructor } from '../BaseEntity';
+
+/**
+ * EntityListMapper transform array of object into entity property with array of entities.
+ */
+export class EntityListMapper extends BaseMapper {
+  #entityMapper: EntityMapper;
+
+  /**
+   * Creates a mapper with the specified entity that is used for the transformation field contains array of object.
```

Comment:
"...for the transformation of a field containing array of objects."

Reply:
Upraveno